styles/Using.module.css

```diff
@@ -0,0 +1,31 @@
+/* @author Charles Liu */
+
+.Container {
+  padding: 0px 5%;
+}
+
+.Row {
+  padding: 50px 0px;
```

Comment:
`py-[1-5]` is a Bootstrap class for Vertical Padding if you need it in the future :)

Reply:
ahh thanks. I wasn't sure if you should use standard css or bootstrap classes bc I was gonna use css modules anyway. I'll keep that in mind for next time